content/overview/workflow/process.md

```diff
@@ -0,0 +1,209 @@
+
+Please note that the [white balance](../../module-reference/processing-modules/white-balance.md) module is still enabled in this approach, but its settings should not usually be altered.
+
+---
+Set [preferences > processing > auto-apply chromatic aberration defaults](../../preferences-settings/processing.md) to "modern" now.
+
+---
+## edit in a controlled environment
+
+Image processing should be performed in a controlled environment, lit by a white light source against a background approximating mid-gray, and on a monitor that has been properly calibrated.
+
+While this may not be practical in many home editing environments you _can_ control the background colors on your monitor's display. You should set the darktable color scheme to use one of the "grey" themes and use the [color assessment mode](../../module-reference/utility-modules/darkroom/color-assessment.md) when altering tones and colors in your image. Dark themes might look good but, unless you are processing images to be viewed on a cinema screen in a darkened room, they should not be used for photo processing.
+
+---
+Set [preferences > general > theme](../../preferences-settings/general.md) to "darktable-elegant-grey" or "darktable-icons-grey" now.
+
+---
+## enter the darkroom
+
+Choose an image to edit from the lighttable view and double-click to load that image into the darkroom view. For now try to choose an image that is well exposed -- we will discuss some techniques to recover badly-exposed images later.
+
+In the darkroom view, you will see a list of [processing modules](../../darkroom/processing-modules/_index.md) to the right of your image. Each module performs its own processing on the image, in the order shown in the module list, starting at the bottom of the list and moving up to the top. You can think of this like a stack of building blocks where each block builds on the processing performed by the modules below it.
+
+On the left hand side is the [history stack](../../module-reference/utility-modules/darkroom/history-stack.md) (you may need to expand the module), which shows the order in which adjustments were made to the controls of the various modules. This allows you to undo changes by reverting to an earlier step in the history stack. You will see that a number of modules are applied automatically -- these are needed in order to generate a legible image from the Raw data.
+
+To the top right is the [histogram](../../module-reference/utility-modules/shared/histogram.md) which shows the spread of tones/colors in your image.
+
+---
+If you have previously viewed or edited the image in the darkroom view, start by _discarding history_ (click the _reset_ button in the [_history stack_](../../module-reference/utility-modules/darkroom/history-stack.md)). This will reapply defaults using your new settings and provide a clean starting point for editing.
+
+---
+## why doesn't the raw image look like the JPEG?
+
```

Comment:
I proposed some illustrative images for this in another PR...

Reply:
I know. I wasn't sure whether they were worth including. The problem is that, with the scene-referred workflow settings, often the default image isn't that bad. Images have to be _really_ compelling to include and I would expect ppl to be viewing this guide with darktable running.